### ISSUE TITLE
--version output should be reproducible

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#554]
In GitLab by @das-g on Jan 13, 2017, 05:26

*Please describe how you installed Flake8*

```bash
mktmpenv -p $(which python3)
pip install flake8
```

*Please provide the exact, unmodified output of `flake8 --bug-report`*
```json
{
  "dependencies": [
    {
      "dependency": "setuptools",
      "version": "32.3.1"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.5.2",
    "system": "Linux"
  },
  "plugins": [
    {
      "plugin": "mccabe",
      "version": "0.5.3"
    },
    {
      "plugin": "pycodestyle",
      "version": "2.2.0"
    },
    {
      "plugin": "pyflakes",
      "version": "1.3.0"
    }
  ],
  "version": "3.2.1"
}
```

*Please describe the problem or feature*

The output of `flake8 --version` differs from invocation to invocation. (The order in which the versions of the included tools are listed doesn't seem to be fixed.) For quickly comparing versions between different machines or different virtualenvs etc. (be it by eye or scripted), it'd be good if the order was deterministic and reproducible. (E.g. alphabetical by tool name.)

*If this is a bug report, please explain with examples (and example code) what you expected to happen and what actually happened.*

Actual:

```
$ flake8 --version
3.2.1 (pyflakes: 1.3.0, mccabe: 0.5.3, pycodestyle: 2.2.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pycodestyle: 2.2.0, pyflakes: 1.3.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (pycodestyle: 2.2.0, mccabe: 0.5.3, pyflakes: 1.3.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pyflakes: 1.3.0, pycodestyle: 2.2.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (pycodestyle: 2.2.0, pyflakes: 1.3.0, mccabe: 0.5.3) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (pyflakes: 1.3.0, pycodestyle: 2.2.0, mccabe: 0.5.3) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (pycodestyle: 2.2.0, mccabe: 0.5.3, pyflakes: 1.3.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pycodestyle: 2.2.0, pyflakes: 1.3.0) CPython 3.5.2 on Linux
```

Expected / wanted:

```
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pycodestyle: 2.2.0, pyflakes: 1.3.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pycodestyle: 2.2.0, pyflakes: 1.3.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pycodestyle: 2.2.0, pyflakes: 1.3.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pycodestyle: 2.2.0, pyflakes: 1.3.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pycodestyle: 2.2.0, pyflakes: 1.3.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pycodestyle: 2.2.0, pyflakes: 1.3.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pycodestyle: 2.2.0, pyflakes: 1.3.0) CPython 3.5.2 on Linux
$ flake8 --version
3.2.1 (mccabe: 0.5.3, pycodestyle: 2.2.0, pyflakes: 1.3.0) CPython 3.5.2 on Linux
```

The output should only change if at least one of the involved versions changed.

**Comment by asottile** [comment]
In GitLab by @das-g on Jan 13, 2017, 06:30

mentioned in merge request !164

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jan 21, 2017, 12:16

mentioned in merge request !169

**Comment by asottile** [comment]
In GitLab by @das-g on Jan 21, 2017, 12:20

closed via commit f0f2ea7f4e3e536f2507703e350c89d0468feb31

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Mar 16, 2017, 12:48

mentioned in commit asottile/flake8@c1f9dc724197a30a4c3d78883138f2b5b82a874e
